packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx

```diff
@@ -178,22 +182,27 @@ const Question = (props: QuestionProps) => {
 
   if (!answerUI || !questionText || !style) return null;
 
+  const hasVideoOrImage =
+    answerUI.media?.type && [TYPE_VIDEO, TYPE_IMAGE].includes(answerUI.media.type);
+
   return (
     <>
       <View style={style.questionHeading}>
-        <Text style={style.questionOrigin}>{questionOrigin}</Text>
-        <Text style={style.questionText}>{questionText}</Text>
-        <Text style={style.questionHelp}>{get('help', answerUI)}</Text>
+        <Html style={style.questionOrigin}>{questionOrigin}</Html>
```

Comment:
any reason for moving from Text to HTML ?

Reply:
these portions of question may be formated as html when defined on cockpit